training/src/main/scala/com/airbnb/aerosolve/training/Evaluation.scala

```diff
@@ -87,6 +88,95 @@ object Evaluation {
       .toArray
   }
 
+  def evaluateBinaryClassification(records : RDD[EvaluationRecord],
+                                   buckets : Int,
+                                   evalMetric : String,
+                                   resultsOutputPath: String) : Array[(String, Double)] = {
+    var metrics  = mutable.Buffer[(String, Double)]()
```

Comment:
lots of code are dup with the other evaluateBinaryClassification, can you share code between them?